code/modules/clothing/shoes/clown.dm

```diff
@@ -61,3 +62,10 @@
 	icon_state = "ducky_shoes"
 	inhand_icon_state = "ducky_shoes"
 	squeak_sound = list('sound/effects/quack.ogg'=1) //quack quack quack quack
+
+/obj/item/clothing/shoes/clown_shoes/green_jester_shoes
```

Comment:
should be moved to the monkestation folder

Reply:
Not completely sure what this means I'm kind of new to this

Reply:
nevermind I know now!

Reply:
I THINK I HAVE FINALLY FINISHED THIS